actionpack/lib/action_dispatch/middleware/templates/rescues/missing_exact_template.html.erb

```diff
@@ -0,0 +1,23 @@
+<header>
+  <h1>Missing exact template</h1>
```

Comment:
"exact" sounds a bit odd in a user-facing context here. How about we title it "No template for interactive request"?